unit_tests/integration/test_external_backtrace_service.py

```
@@ -0,0 +1,55 @@
+
+    assert result, "Decoded output should not be empty"
+    assert "Backtrace" in result
+    assert any(indicator in result for indicator in ["at ", "()", "::", "FRAME"]), (
+        f"Decoded output doesn't look like resolved symbols: {result[:200]}"
+    )
+
+
+def test_decode_via_external_service_raises_for_unknown_build(node):
+    """Verify _decode_via_external_service raises for an unknown build ID."""
+    fake_build_id = "0" * 40
+    with pytest.raises(Exception):
```

Ruff (0.15.14)

[warning] Do not assert blind exception: `Exception`

(B017)